fail2ban/server/filter.py

```diff
@@ -360,11 +360,11 @@ def inIgnoreIPList(self, ip):
 				s.insert(1, '32')
 			elif "." in s[1]: # 255.255.255.0 style mask
 				s[1] = len(re.search(
-					"(?<=b)1+", bin(DNSUtils.addr2bin(s[1]))).group())
+					"(?<=b)1+", bin(DNSUtils.addr2dec(s[1]))).group())
 			s[1] = long(s[1])
 			try:
-				a = DNSUtils.cidr(s[0], s[1])
-				b = DNSUtils.cidr(ip, s[1])
+				a = DNSUtils.addr2dec(s[0], cidr=s[1])
+				b = DNSUtils.addr2dec(ip, cidr=s[1])
```

Comment:
please keep it 'addr2bin' for now -- I have replied to outdated now diff - so not sure if reached